sdcm/tester.py

```diff
@@ -2854,6 +2860,10 @@ def tearDown(self):
         self.stop_event_device()
         if self.params.get('collect_logs'):
             self.collect_sct_logs()
+        cleanup_ssl_config()
+        # Revert cqlshrc to original state
+        update_cqlshrc.cache_clear()
```

Comment:
why clear this, and why update the cqlsh file at the end of the test ? not sure what do we gain by doing so ?

Reply:
Cleaning up is to not commit SSL artifacts that were created at the beginning of the test - CA certificates, keys, Java keystore - so that a person won't need to remove them manually, if works with SCT locally.

The same for cqlshrc - if a test configuration enables hostname validation, we rollback the attribute value after the test, to not commit the change.

Reply:
we might want to limit this to when it running locally, in CI it's just a waste of time 

we should just add those to .gitignore from this change onwards